marklogic-data-hub-central/ui/e2e/cypress/integration/scenarios.json.tsx

```diff
@@ -325,4 +325,27 @@ describe('json scenario for table on browse documents page', () => {
     detailPage.getDocumentJSON().should('exist');
   });
 
+  it('search for multiple facets, switch to snippet view, delete a facet, switch to table view, verify search query', () => {
+    browsePage.selectEntity('Customer');
+    browsePage.getSelectedEntity().should('contain', 'Customer');
+    cy.wait(500);
+    browsePage.getFacetItemCheckbox('firstname', 'Kelley').click();
+    browsePage.getFacetItemCheckbox('lastname', 'Oneal').click();
+    browsePage.getSelectedFacets().should('exist');
+    browsePage.getGreySelectedFacets('Kelley').should('exist');
+    browsePage.getFacetApplyButton().should('exist');
+    browsePage.getFacetApplyButton().click();
```

Comment:
Do we have to assert that apply button exist before hitting click. click action would wait for the element right. https://docs.cypress.io/guides/core-concepts/interacting-with-elements.html#Visibility

Reply:
I removed the extra check for the apply button